smalld/smalld.py

```diff
@@ -89,6 +95,29 @@ def run(self):
             time.sleep(5)
 
 
+class Intent(Flag):
+    # Gateway intents https://discord.com/developers/docs/topics/gateway#gateway-intents
+    GUILDS = auto()
```

Comment:
I'm not sure we should depend up auto here, since the flag values are defined by someone who isn't us.